ansible/roles/airflow/templates/airflow.cfg.j2

```diff
@@ -0,0 +1,350 @@
+[core]
+# The home folder for airflow, default is ~/airflow
+airflow_home = {{ airflow_home }}
+
+# The folder where your airflow pipelines live, most likely a
+# subfolder in a code repository
+# This path must be absolute
+dags_folder = {{ airflow_home }}/dags
+
+# The folder where airflow should store its log files
+# This path must be absolute
+base_log_folder = {{ airflow_home }}/logs
+
+# Airflow can store logs remotely in AWS S3 or Google Cloud Storage. Users
+# must supply a remote location URL (starting with either 's3://...' or
+# 'gs://...') and an Airflow connection id that provides access to the storage
+# location.
+remote_base_log_folder =
+remote_log_conn_id =
+# Use server-side encryption for logs stored in S3
+encrypt_s3_logs = False
+# DEPRECATED option for remote log storage, use remote_base_log_folder instead!
+s3_log_folder =
+
+# The executor class that airflow should use. Choices include
+# SequentialExecutor, LocalExecutor, CeleryExecutor
+executor = LocalExecutor
+
+# The SqlAlchemy connection string to the metadata database.
+# SqlAlchemy supports many different database engine, more information
+# their website
+sql_alchemy_conn = postgresql://{{ postgres_users.commcare.username }}:{{ postgres_users.commcare.password }}@127.0.0.1:5432/airflow
```

Comment:
db host should be parameterized and port should be 6432 (or parameterized)

Possibly take the same approach as formplayer: https://github.com/dimagi/commcare-cloud/blob/760932dcebaa8e31bfd82fb7c8e4a53d700dad97/ansible/roles/formplayer/templates/application.properties.j2#L31